src/trace/context.ts

```diff
@@ -58,15 +63,24 @@ export function readTraceFromEvent(event: any): TraceContext | undefined {
   if (typeof headers !== "object") {
     return;
   }
-  const traceID = headers[traceIDHeader];
+
+  const lowerCaseHeaders = Object.keys(headers).reduce(
```

Comment:
Just a nits/personal preference, feel free to ignore.

I feel `map` makes it slightly easier to read than `reduce`, which normally reduces an object/array to a single value?

```javascript
  const lowerCaseHeaders = {};
  Object.keys(headers).map(key => {lowerCaseHeaders[key.toLocaleLowerCase()]= headers[key]})
```

Reply:
Since it isn't returning anything in the map function, it's better to just use a for loop. I've updated this.